docker/mem0/main.py

```
@@ -0,0 +1,329 @@
+    params = {
+        k: v
+        for k, v in memory_create.model_dump().items()
+        if v is not None and k != "messages"
+    }
+    try:
+        response = MEMORY_INSTANCE.add(
+            messages=[m.model_dump() for m in memory_create.messages], **params
+        )
+        return JSONResponse(content=response)
+    except Exception as e:
+        logging.exception("Error in add_memory:")
```

Ruff (0.15.6)

[warning] `exception()` call on root logger

(LOG015)